
def test_error():
    raise ValueError('This is an error to test pytest in GitHub Actions.')
